File: C++ New/01_Basic/Lec_02/pattern3.cpp
//Right angled triangle of fibbonacci number
#include <iostream>
using namespace std;
int main(){
    int row;
    cout << "Enter the number of rows : ";
    cin >> row;
    int a = 0;
    int b = 1;
    for(int i = 0; i < row; i++){
        for(int j = 0; j <= i; j++){
            cout << a << " ";
            int c = a + b;
            a = b;
            b = c;
        }
        cout << endl;
    }
    return 0;
}

File: JAVA/01_core_JAVA/Lecture02/pattern3.java
//Code-Right angled triangle from fibonacci series
package Lecture02;

import java.util.Scanner;

public class pattern3 {

	public static void main(String[] args) {
		Scanner scn = new Scanner(System.in);
		System.out.print("Please Enter a number: ");
		int n = scn.nextInt();
		int a = 0, b = 1;
		for (int i = 1; i <= n; i++) {
			for (int j = 1; j <= i; j++) {
				System.out.print(a + "  ");
				int c = a + b;
				a = b;
				b = c;
			}
			System.out.println();
		}
	}

}
